src/pages/Integrations/Create/helpers.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import type {
+  IntegrationIcon,
+  IntegrationIconTypes,
+} from '../../../types/Integration';
+
+export const iconMapper =
+  (integrationTypes: IntegrationIconTypes | undefined) =>
+  (name: string): React.FunctionComponent | null => {
+    if (!integrationTypes) {
+      return null;
+    }
+
+    const integrationType: IntegrationIcon | undefined = Object.values(
+      integrationTypes
+    ).find((type: IntegrationIcon) => type.name === name);
+
+    if (!integrationType) {
+      return null;
+    }
+
+    const Icon = () => (
+      <img
+        src={integrationType.icon_url}
+        alt={integrationType.product_name}
+        className="src-c-wizard__icon pf-u-mb-sm"
+      />
+    );
+
+    return Icon;
+  };
+
+export const compileAllIntegrationComboOptions = (
```

Comment:
This feels a bit cumbersome, we already have all available integrations, let's map their names to labels.

Reply:
No need to add extra work to this PR as majority of it will be moved o sources anyway.